backend/src/EntityExtractor.Contracts/IFileStoring.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace eKultura.EntityExtractor.Contracts;
+
+public interface IFileStoring
+{
+    FileStoringDTO StoreFile(string topic, byte[] pdfFile);
```

Comment:
I/O operations are generally slow and synchronous method would block the thread. Please change it to asynchronou version